index.html

```diff
@@ -2326,6 +2328,12 @@ <h3 class="heading settled" data-level="11.3" id="incoming-stream-methods"><span
   STOP_SENDING frame) with its error code set to the value of
   |abortInfo.errorCode|.</p>
      </ol>
+    <dt data-md><dfn class="dfn-paneled idl-code" data-dfn-for="IncomingStream" data-dfn-type="method" data-export id="dom-incomingstream-arraybuffer"><code>arrayBuffer()</code></dfn>
+    <dd data-md>
+     <p>A convenience method that asynchronously reads all the contents of
+ |IncomingStream.readable| and returns it as an ArrayBuffer.  This locks the
+ stream while reading, just as if |IncomingStream.readable.getReader()| were
+ used.</p>
```

Comment:
I like this convenience method, but did you intent do put it in this PR?  It isn't mentioned in the PR description.

Reply:
Wouldn't we want IncomingStream to include the Body mixin for full features?

Reply:
Apparently this was added [here](https://github.com/WICG/web-transport/commit/9256880c592dcef4dfb4a8730ad3fb8a045991f8), but index.html wasn't updated until now.